visuals/js/index.js

```
@@ -109,9 +118,9 @@ function toggleEndScreen(data) {
   }
 }
 
-function updateMoveCount(moveCount) {
-  document.querySelector(".move_number").textContent =
-    "Move: " + (moveCount || "####");
+function updateMoveCount(generationCount) {
+  document.querySelector(".generation_number").textContent =
+    "Generation: " + (generationCount || "####");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)